packages/terra-image/src/Image.jsx

```diff
@@ -61,7 +61,7 @@ const propTypes = {
   /**
   * Sets the `object-fit` style of the image from the following values; `cover`, `contain`, `fill`, `scale-down`, `none`.
   */
-  fit: PropTypes.oneOf(Object.values(FitTypes)),
+  fit: PropTypes.oneOf(Object.values(FitTypes)), // eslint-disable-line compat/compat
```

Comment:
Can we add settings to our eslint configuration to prevent adding these compat disables in the code base? https://github.com/amilajack/eslint-plugin-compat#v3

Reply:
👍 Updated with these commits:
* https://github.com/cerner/terra-core/pull/2581/commits/74c51bc158387ed1e0d32958ab732615e84f8976
* https://github.com/cerner/eslint-config-terra/pull/34/commits/945ea3333cb10512005ecdddfcec559c34a172c6
* https://github.com/cerner/eslint-config-terra/pull/34/commits/567a0b39914720ff32c187fed01e5bb5b598748b